### ISSUE TITLE
compute-val-loss

### ISSUE THREAD

**Issue by rithvikrthampy** [fizyr/keras-retinanet#1328]
How to use compute-val-loss

**Comment by hgaiser** [comment]
`retinanet-train --help`
